yarn.lock

```diff
@@ -8449,7 +8449,7 @@ write-file-atomic@^3.0.0:
     signal-exit "^3.0.2"
     typedarray-to-buffer "^3.1.5"
 
-ws@^5.0.0, ws@^5.2.0:
+ws@^5.2.0:
   version "5.2.2"
```

Comment:
What uses this dep? We have the explicit 7.4.6 but there is an implicit 5.2.0 somewhere

Reply:
![image](https://user-images.githubusercontent.com/42871571/120469875-96195a80-c3ab-11eb-973b-ecd0a6c5ab03.png)

Reply:
So apollo server also uses it and there's no remediation 🤦🏼

Reply:
yap
the only action item was upgrading the direct dependency of us.

and btw - although everything works perfect on my local machine, the tests break because they cannot detect explorook is up

Reply:
![image](https://user-images.githubusercontent.com/42871571/120470935-cd3c3b80-c3ac-11eb-948b-6577fda7bc6e.png)

Reply:
Maybe it's a problem with the headless start. Try reading the build logs, there might even be some logs in the assets

Reply:
The fe-facade is also using the same apollo-server version but in its yarn.lock it uses ws v7